index.js

```diff
@@ -12,48 +12,53 @@ module.exports.decodeBox2 = function(box2) {
 // assumes msg has already been validated
 // returns a classic compatible json object
 module.exports.decode = function(bmsg) {
-  const decoded = bencode.decode(bmsg)
+  const [payload, signature] = bencode.decode(bmsg)
```

Comment:
Ooh this destructuring is clean; great. Makes other references to these values much clearer.